core/services/bulletprooftxmanager/eth_confirmer.go

```diff
@@ -326,7 +341,7 @@ func (ec *ethConfirmer) batchFetchReceipts(ctx context.Context, attempts []model
 	return
 }
 
-func (ec *ethConfirmer) saveFetchedReceipts(ctx context.Context, receipts []Receipt) (err error) {
+func (ec *ethConfirmer) saveFetchedReceipts(receipts []Receipt) (err error) {
```

Comment:
It doesn't seem like a good idea to stop threading contexts through all of these functions, even if they're unused currently -- even if only because of newcomers to the team and the example that we're providing to them

Reply:
The reason for this is because I am not sure it makes sense to cancel database requests on shutdown - ideally we would want those to complete cleanly (although the data model ought to be robust to unexpected rollbacks).